api/controllers/PostController.js

```diff
@@ -58,10 +58,6 @@ const findTagId = req =>
   req.param('tag') && Tag.find(req.param('tag')).then(t => t.id)
 
 const queryForCommunity = function (req, res) {
-  if (TokenAuth.isAuthenticated(res)) {
-    if (!RequestValidation.requireTimeRange(req, res)) return
```

Comment:
Yes and no. This was something I put in place not only because Harlan asked for it, but because it would avoid the default behavior being "fetch everything for a community", to reduce impact on the server. We don't have to do it this way, but it is something we should think about. Maybe a default value for `limit` would do as well.